docs/installation/upgrade/2.3-to-2.4.md

```diff
@@ -21,6 +21,44 @@ setup.
 
 ## Dovecot CE
 
+### Configuration Changes
+
+::: warning
+Dovecot 2.3.x settings will NOT work unless the configuration is changed
+as described in this section.
```

Comment:
This is missing all the important notes about configuration being completely redesigned. Not sure what exactly to write there, but some note at a minimum.